_includes/02-image.md

```diff
@@ -0,0 +1 @@
+Replace this with an image, like your photo. Ensure you include some alt-text.
```

Comment:
![Image of Red Rose](https://www.pexels.com/photo/beautiful-bloom-blooming-close-up-593655/)